wealthy-senior-execs/config/_typography.scss

```diff
@@ -6,6 +6,7 @@ $secondary-font: Vollkorn, 'Liberation Serif', Times, 'Times New Roman', serif;
 
```

Comment:
I know none of this is changed here, but a few things I noticed:

- These font stacks don't match those specified:
>Primary – Roboto, Helvetica Neue, gotham, helvetica, arial, sanserif
>Secondary – Volkorn, Cambria, Hoefler text, liberation serif, times, times new roman, serif
- Do we want to set a mixin for secondary-font so that it uses the correct font weight as standard, rather than relying on people setting it correctly whenever they use it? We also need to load in this font weight, currently we are loading 400 instead of 500?
- We probably want to load in 700 for Roboto too as we'll probably want bold body copy at some point!
- Not sure if we need to load in the italic and bold-italic variants too, or if these will work without doing?

Reply:
I _think_ I've done these correctly now but would be good to get a sense check!